andela/ah-shakas-frontend
# #160577435 Setup SASS processing

#### What does this PR do?
- Setups processing for SASS to allow you to use `.sass` and `.scss` files that will be compiled to css

#### Description of Task to be completed?

- Add sass for the node project by running `yarn add node-sass`.
- Changed the `.css` files to `.scss` and changed the imports to reflect this change.
- Added  `.css` to `.gitignore`

#### How should this be manually tested?

- Clone the github repository `git clone https://github.com/andela/ah-shakas-frontend` on your machine.
- Change your working directory to the project folder `cd ah-shakas-frontend`
- Move to the relevant branch by `git checkout ch-sass-160577435`
- Install the required dependencies using `yarn install`
- Run the local development server using `yarn start` and you should get the following message:
  ```
  Compiled successfully!

  You can now view ah-shakas-frontend in the browser.
  
    Local:            http://localhost:3000/
    On Your Network:  http://10.10.160.30:3000/
  
  Note that the development build is not optimized.

#### Any background context you want to provide?

- SASS allows you to use both `.scss` and `.sass` extentions that can be compiled into `.css`

#### What are the relevant pivotal tracker stories?
[#160577435](https://www.pivotaltracker.com/n/projects/2198533/stories/160577435)
#### Screenshots (if appropriate)
N/A
